### ISSUE TITLE
plot a scatter chart on a bar chart with same bar values 

### ISSUE THREAD

**Issue by dev-1603** [gitbrent/PptxGenJS#674]
1. i want to plot a scatter chart on a stacked bar ,  so i want to know is there any way possible to calculate scatter points based on bar chart  values ?
what is the  conversion rate  for  scale conversion or chart and scatter chart. right now it not properly aligned.
 2. can we add waterfall chart to our slide ??? 
![ppt scatter](https://user-images.githubusercontent.com/53967786/74525872-3134b980-4f48-11ea-8add-dbe5569b6e24.JPG)

**Comment by gitbrent** [comment]
1. I'm not sure 
2. Waterfall charts are not supported as of yet

**Comment by dev-1603** [comment]
Hi Brent ,
i think  i am not able to explain the issue  so let me try again ,
 as pptxGENJS doesn't support waterfall charts so  this is what i am doing
 to create  a waterfall charts:
 1. creating a stacked bar chart .
2. setting the color of lower stacked bar  same as  the background ,so its
not visible now , i am disabling the bar labels because  i want to print
the values of visible bars .
3. After that i am plotting the scatter chart on the same chart  . there i
am setting  points height as(value of red + value of blue +1)   [referring
your example ]
[image: image.png]

Now the problem  which i am facing is i a don't know the scale which you
are using to plot scatter point chart ,
so can you tell me if  i plot scatter point x value ( referring chart 1 )
of April  as   17+55+28 will it be plotted at 100??
same as for y value in  chart 2 .


this is what i tried  in the graph below  but i am not getting  desired
output  as you you can see the space between scatter point  label and
stacked bar is increasing  from top to bottom
[image: ppt scatter.JPG]
if you could figure out any solution for this .. then it will  be a great
help
will be waiting for your response
thank you
